### ISSUE TITLE
Decoder not saving Ambisonics Order in Reaper

### ISSUE THREAD

**Issue by IokaCode** [schweizerweb/icst-ambisonics-plugins#7]
When reopening a Reaper session, the Decoder will not recall the last selected Ambisonics order but will default back to 1. order.

**Comment by christian-schweizer** [comment]
Thanks for reporting, we'll have a look at this! Can you confirm it's the case for the DECODER only?

**Comment by IokaCode** [comment]
Yes it seems to be the case only for the Decoder. I checked the output metering of the Encoders and this looked good everytime.
Only with the Decoder the higher orders are missing when reopening a project as can be seen on the screenshot:
![grafik](https://github.com/schweizerweb/icst-ambisonics-plugins/assets/87419514/aeaffa20-e932-48f8-8174-4d8b0c7863d1)

**Comment by schweizerweb** [comment]
Fixed in Version 3.1.0
